manifests/params.pp

```diff
@@ -17,7 +17,9 @@
   $package_name   = undef
   $package_ensure = 'present'
   $group_id       = undef
+  $system_group   = true
   $user_id        = undef
+  $system_user    = true
```

Comment:
Since I don't want the old behavior to change, I am going to change these new `system_`-variables to `false`.